### ISSUE TITLE
Modular Storage rotates with wheat in your hand

### ISSUE THREAD

**Issue by ivardensphere** [McJty/RFTools#197]
Sort of a strange bug but wheat seems to be acting like a wrench and rotates the Modular Storage. Using the latest version.

**Comment by firecracker37** [comment]
I came here to report the same thing.  Also gunpowder seems to have the same effect.

**Comment by 4rz0** [comment]
Also happens with gold (nuggets and ingots).

**Comment by McJty** [comment]
Hmm. I cannot reproduce this... Must be an interaction with some other mod. Weird...

**Comment by 4rz0** [comment]
I'll do some experimenting once I'm at home with the mods we have installed.
Thing is, you can only rotate the RFTools machines, not BC or TE ones for example.

**Comment by McJty** [comment]
I can reproduce it in my own modpack but not in my dev environment. It is really weird. I see nothing in the code that would allow this rotation for things like wheat.

**Comment by 4rz0** [comment]
Maybe it's easier to find if we can find the mod that causes this interaction glitch.

**Comment by McJty** [comment]
At RFTools gives a number of messages like this in the log:

Failed to load wrench class...

Could you tell me which wrenches it couldn't load (and so I can see which ones could be loaded and it is trying to use).

**Comment by 4rz0** [comment]
I'll report back as soon as I find something.

**Comment by 4rz0** [comment]
```
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class resonant.core.content.ItemScrewdriver (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class ic2.core.item.tool.ItemToolWrench (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class ic2.core.item.tool.ItemToolWrenchElectric (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class com.bluepowermod.items.ItemScrewdriver (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class crazypants.enderio.api.tool.ITool (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class mekanism.api.IMekWrench (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class pneumaticCraft.common.item.ItemPneumaticWrench (this is not an error)
[19:34:11] [Server thread/INFO] [mcjty.rftools.RFTools/rftools]: Failed to load wrench class powercrystals.minefactoryreloaded.api.IToolHammer (this is not an error)
```

I should note that we in fact have MFR installed, but the admin pointed out that the correct class for those wrenches seems to be `powercrystals.minefactoryreloaded.api.IMFRHammer`.

I know of following wrench-type tools installed on our server:
* Buildcraft-Wrench
* Redstone Arsenal OnmiWrench
* Redstone Arsenal BattleWrench
* Thermal Expansion Crescent Hammer
* Thermal Expansion BattleWrench
* Applied Energistics 2 Wrench (Certus-/Netherquartz)
* RFTools wrench (of course)
* Railcraft Crowbar (however, this one doesn't seem to work on RFTools machines)

**Comment by Timeslice42** [comment]
I get the same behavior with the Crafter. Mods with wrenches installed:
GalacticraftCore - Standard Wrench
Thermal Expansion - Crescent Hammer
Thermal Expansion - BattleWrench
Applied Energistics 2 = Certus/Nether Quartz Wrench, Network Tool
RFTools - Smart Wrench
MineFactory Reloaded - Precision Sledgehammer

and technically a stick doubles as a wrench for most Extra Utilities blocks, but I don't expect that that would count.

**Comment by McJty** [comment]
Fixed in 2.40!
